source/dynamical_systems/README.md

```diff
@@ -117,11 +120,11 @@ state_representation::CartesianTwist twist = linear.evaluate(csB);
 
 The returned velocity will always be expressed in the same reference frame as the input state.
 
-### Reference frames
+### Reference frames (this applies to all of them, also a note on empty construction?!)
```

Comment:
We have this paragraph here explicitly under Linear but it actually applies to all of the Cartesian DS's so I was wondering if we should maybe reorganize this a bit

Reply:
I would suggest moving this section to right after `Base DynamicalSystem-> Base frame`, as that will keep it general and also has a nice continuation from the end of the base frame section.

You can remove the `See the section on Linear DS reference frames for examples.` from the end of the base frame section. Then edit the first line of this Reference frames section to be:
```
The following section applies to derived DS classes using the CartesianState type. The [Linear DS](#link linear ds) will be used as an example.
```

You will have to adjust the table of contents as well.